src/app/pages/virtrolio-cover/virtrolio-cover.component.html

```diff
@@ -66,7 +66,7 @@ <h5 class="modal-title" id="link-gen-title">Ahoy, your sharing link has been pre
             new link
           </button>
           <div id="warning-div" *ngIf="showWarningText">
-            Are you sure? Generating a new link will invalidate all older ones! Click generate again to confirm.
+            Are you sure? Generating a new link will invalidate your last one! Click generate again to confirm.
```

Comment:
```suggestion
            Are you sure? Generating a new link will invalidate any other links that you have generated! Click generate again to confirm.
```

Reply:
@ricetech but they only have one link at at time. I didn't want to confuse them into thinking they could have more than one at a time.